model-engine/model_engine_server/common/constants.py

```diff
@@ -2,6 +2,8 @@
 
 BILLING_POST_INFERENCE_HOOK: str = "billing"
 CALLBACK_POST_INFERENCE_HOOK: str = "callback"
+LOGGING_POST_INFERENCE_HOOK: str = "logging"
+LOGGING_FIREHOSE_STREAM: str = "PUT-ML-LOGGING-TEST-1"
```

Comment:
guessing this is just for testing/not used in production?

Reply:
for production as well, will move this to infra config